canonical-web-and-design/snapcraft.io
# 2768 add tests for builds

## Done

- Add some tests for buils

## Issue / Card

Fixes #2768 

## QA

- Pull the branch
- Run the site using the command `./run`
- View the site locally in your web browser at: http://0.0.0.0:8004/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Run `dotrun exec yarn test-js` locally and all js tests should pass
